docs/game_data/spel2.lua

```diff
@@ -1321,8 +1321,8 @@ function get_short_tile_code_definition(short_tile_code) end
 ---Set `is_valid` to `nil` in order to use the default rule (aka. on top of floor and not obstructed).
 ---If a user disables your script but still uses your level mod nothing will be spawned in place of your procedural spawn.
 ---@param procedural_spawn string
----@param do_spawn fun(): any
----@param is_valid fun(): any
+---@param do_spawn function
```

Comment:
And adjust the comments that are not caught by your new logic?

Reply:
I probably would have to add something different when there's multiple callbacks in a single function.